src/models/disassemblyoutput.cpp

```diff
@@ -272,7 +272,9 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
     const auto processPath = QStandardPaths::findExecutable(objdump);
     if (processPath.isEmpty()) {
         disassemblyOutput.errorMessage =
-            QApplication::tr("Cannot find objdump process %1, please install the missing binutils package for arch %2")
+            QApplication::translate("DisassemblyOutput",
```

Comment:
these hunks here are unrelated to what the commit message talks about - split that up into a separate commit